suite-native/module-settings/src/slice.ts

```diff
@@ -23,12 +25,17 @@ export const appSettingsSlice = createSlice({
         setColorScheme: (state, action: PayloadAction<AppColorScheme>) => {
             state.colorScheme = action.payload;
         },
+        setOnboardingFinished: (state, action: PayloadAction<boolean>) => {
+            state.isOnboardingFinished = action.payload;
+        },
```

Comment:
Seems like you're not using this action at all

Reply:
I am using it https://github.com/trezor/trezor-suite/pull/6171/files#diff-44f0ef7af9c05124bb46149eaf337105c82cfec1b0a873558d9d97f9f08932f5R109